app/models/itinerary.rb

```diff
@@ -0,0 +1,9 @@
+class Itinerary < ApplicationRecord
+  has_many :itinerary_flights
+  has_many :flights, through: :itinerary_flights
+  has_many :itinerary_passengers
+  has_many :passengers, through: :itinerary_passengers
+  has_one :invoice
```

Comment:
I think this is missing a `belongs_to :user

Reply:
Done! updated as we discuss